test/MiniCodec.test.js

```
@@ -0,0 +1,68 @@
+        console.log(MiniCodec.encodeString("lol"))
+        return [
+            user.id,
+            ...MiniCodec.encodeString(user.name),
+            user.profile.age,
+            ...MiniCodec.encodeString(user.profile.email),
+            ...MiniCodec.encodeString(user.profile.preferences.theme),
+            user.profile.preferences.notifications ? 1 : 0
+        ];
+    },
+    decode(encoded) {
+        let posRef = { pos: 0 };
```

Biome

[error] This let declares a variable that is only assigned once.

'posRef' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)